cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -237,9 +237,10 @@ class OrtMainFunTest : StringSpec() {
                 runMain(
                     "-c", configFile.path,
                     "evaluate",
-                    "-i", "build.gradle.kts",
-                    "--rules-file", "build.gradle.kts",
-                    "--rules-resource", "DUMMY"
+                    "-i", "src/funTest/assets/semver4j-ort-result.yml",
```

Comment:
Have you considered to instead provide a `osadlLicenseRule`, or however one would call it, in ORT's API so that the rules can make use of it, just as it can for all other rules in ORT's API?

I'm asking because that would keep the evaluator simpler. So, is there any particular reason why this is not possible / desired?

Reply:
> Have you considered to instead provide a `osadlLicenseRule`, or however one would call it, in ORT's API

TBH I haven't considered it, I guess because currently the "highest level" of reusable rules code in the ORT API is a `Rule`, but here we'd need a whole `RuleSet`.

> So, is there any particular reason why this is not possible / desired?

I had two independent asks for the evaluator to support multiple scripts recently. It's simply more convenient for users to just refer to those scripts without any need to write own "glue code". Also, separate scripts allow users to maintain them independently, e.g. one for OSS compliance folks and once for security officers to create violations on security vulnerabilities.

Reply:
> Having multiple rule files will makes things hard to maintain as an ORT user

I disagree. There is no breaking change in the user-facing API (i.e. the CLI) here, so everything that worked before will continue to work, and there is no *need* for the user to actually use / maintain multiple rules files.

> prefer to keep things simple

Keeping things simple for the *user* is fine. But I believe we should not recoil from adding a *bit* of complexity on our side if it actually helps our users, like in this case.

> not seeing the use case to introduce this feature.

I guess our comments overlapped. The concrete use-case / ask for this feature is described in my comment above.

Reply:
After the discussion in the community meeting today I agree that there are valid use cases for multiple rule files and that this does not add much complexity as you can keep using a single file if you prefer, so I am fine with this change.